doc/dev/installation.rst

```diff
@@ -2,7 +2,7 @@ Installation
 ============
 
 
-Catalyst is officially supported on Linux (aarch64/arm64, x86_64) and macOS (aarch64/arm64, x86_64) 
+Catalyst is officially supported on Linux (x86_64, aarch64) and macOS (arm64, x86_64) 
```

Comment:
I remember @dime10 mentioning that adding `arm64` might help users not recognizing the `aarch64` tag.

Reply:
@rauletorresc this commit was added by @dime10 in c3df13846046787e1f6918ff08f0de9bad528721